src/main/cpp/blaze.cc

```diff
@@ -715,6 +715,20 @@ static void RunBatchMode(
 
   jvm_args_vector.insert(jvm_args_vector.end(), command_arguments.begin(),
                          command_arguments.end());
+  if (option_processor.UsingParamFile()) {
+    blaze_util::Path param_file =
+      blaze_util::Path(startup_options.output_base).GetRelative("params");
+    string jvm_execve_exe = jvm_args_vector[0];
+
+    string content;
+    for(auto i = jvm_args_vector.begin()+1; i != jvm_args_vector.end(); ++i) {
+      content.length() == 0 ?
+        content = blaze_util::Quoted(*i) :
+        content += '\n' + blaze_util::Quoted(*i);
```

Comment:
The java argument file format takes all whitespace as separate arguments, so we need to appropriately quote each.
https://docs.oracle.com/javase/9/tools/java.htm#JSWOR-GUID-4856361B-8BFD-4964-AE84-121F5F6CF111